Podcast/FeedTableViewCell.swift

```diff
@@ -0,0 +1,460 @@
+        moreButton.frame = CGRect(x: moreButtonX, y: 0, width: moreButtonWidth, height: moreButtonHeight)
+        playButton.center.y = bottomViewHeight / 2
+        moreButton.center.y = bottomViewHeight / 2
+        recommendedLabel.center.y = bottomViewHeight / 2
+        recommendedLabel.frame.origin.x = recommendedButton.frame.origin.x + buttonPadding + recommendedButton.frame.width
+        playLabel.frame = CGRect(x: playButtonX + playButtonWidth + buttonPadding, y: 0, width: 0, height: 0)
+        playLabel.sizeToFit()
+        playLabel.center.y = bottomViewHeight / 2
+        
+        feedControlButton.frame = CGRect(x: feedControlButtonX, y: 0, width: feedControlButtonWidth, height: feedControlButtonHieght)
+        feedControlButton.center.y = contextViewHeight / 2
+
+        lineSeperator.frame = CGRect(x: lineSeperatorX, y: mainViewHeight - 1, width: frame.width - 2 * lineSeperatorX, height: lineSeperatorHeight)
+        topLineSeperator.frame = CGRect(x: 0, y: 0, width: frame.width, height: lineSeperatorHeight)
+        seperator.frame = CGRect(x: 0, y: frame.height - seperatorHeight, width: frame.width, height: seperatorHeight)
+        
+        
+    }
+    
+
+    ///
+    ///Mark - Buttons
+    ///
+    
+    
+    func bookmarkButtonPress() {
+        if let card = card as? EpisodeCard {
+            card.isBookmarked = !card.isBookmarked
+            if card.isBookmarked == true {
+                bookmarkButton.setImage(#imageLiteral(resourceName: "bookmarkFeedIcon"), for: .normal)
+                
+            } else {
+                bookmarkButton.setImage(#imageLiteral(resourceName: "bookmarkFeedIcon_unselected"), for: .normal)
+            }
+        }
+    }
+    
+        
+    func recommendedButtonPress() {
+        if let card = card as? EpisodeCard {
+            card.isRecommended = !card.isRecommended
+            if card.isRecommended == true {
+                recommendedButton.setImage(#imageLiteral(resourceName: "heart_icon_selected"), for: .normal)
+            } else {
+                recommendedButton.setImage(#imageLiteral(resourceName: "heart_icon"), for: .normal)
+            }
+        }
+    }
+    
+        
+    func moreButtonPress() {
+        
+    }
+    
+        
+    func playButtonPress() {
+        playing = !playing
+        if playing {
+            playButton.setImage(#imageLiteral(resourceName: "play_feed_icon_selected"), for: .normal)
+            playLabel.text = "Now Playing"
+            playLabel.sizeToFit()
+        } else {
+            playButton.setImage(#imageLiteral(resourceName: "play_feed_icon"), for: .normal)
+            playLabel.text = "Play"
+        }
+    }
+    
+        
+    func feedControlButtonPress() {
+        
+    }
+    
+    
+    ///
+    ///MARK - setup card 
+    ///
+    
+    //set the variables when card is passed in
+    func setCard() {
+        if let card = card as? EpisodeCard {
```

Comment:
This should be deleted. Instead there should be a function that populated the view based on a model, but doesn't hold onto the model directly